include/mxnet/c_api.h

```diff
@@ -1274,10 +1274,28 @@ MXNET_DLL int MXCreateCachedOpEx(SymbolHandle handle,
                                  const char** keys,
                                  const char** vals,
                                  CachedOpHandle *out);
+
+/*!
+ * \brief create cached operator, allows to choose thread_safe version
+ * of cachedop
+ */
+MXNET_DLL int MXCreateCachedOpEX(SymbolHandle handle,
+                                 int num_flags,
+                                 const char** keys,
+                                 const char** vals,
+                                 CachedOpHandle *out,
+                                 bool thread_safe = false);
```

Comment:
Aren't default arguments like this only supported in C++ and this file is just C?

Reply:
Yes i need to DEFAULT(false) like https://github.com/apache/incubator-mxnet/blob/master/include/mxnet/c_api.h#L3157